docs/source/checks/vision/data_integrity/plot_property_label_correlation.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+"""
+.. _plot_vision_feature_label_correlation:
+
+Property Label Correlation
+**************************
+
+This notebook provides an overview for using and understanding the "Property Label Correlation" check.
+
+**Structure:**
+
+* `What is the purpose of the check? <#what-is-the-purpose-of-the-check>`__
+* `Run the check <#run-the-check>`__
+* `Define a condition <#define-a-condition>`__
+
+What is the purpose of the check?
+=================================
+The check estimates for every image property (such as brightness, contrast etc.)
+its ability to predict the label by itself.
+
+This check can help find a potential bias in the dataset, that might lead to shortcut learning - the labels being
+strongly correlated with simple image properties such as color, brightness, aspect ratio and more. This is a critical
```

Comment:
This sentence's structure is a bit confusing for someone who doesn't know the exact meaning of "shortcut learning".
(It sounds as if shortcut learning == "the labels being strongly correlated with...")
Recommend to restructure these few sentences to help avoid this potential confusion

Reply:
offer - you can consider adding a link to explanation about shortcut learning if there's a good one (mainly if it's helpful so the user understand this is a type of problem and here they can learn more about it)